samples/matter/common/src/bridge/bridged_device.h

```diff
@@ -71,6 +71,8 @@ class BridgedDevice {
 	CHIP_ERROR HandleReadDescriptor(chip::AttributeId attributeId, uint8_t *buffer, uint16_t maxReadLength);
 
 	bool GetIsReachable() { return mIsReachable; }
+	void NotifyLost() { mIsReachable = false; }
```

Comment:
I can see for now these two functions are not used, however anyway I don't think we should directly modify this state. I would recommend creating a new method of BLEBridgedDeviceProvider that would trigger calling NotifyUpdateState for the IsReachable attribute. It should be propagated to the proper BridgedDevice through the BridgeManager (the IsReachable attribute parsing is not yet supported in our example devices)